pennylane_orquestra/orquestra_device.py

```diff
@@ -118,8 +118,8 @@ class OrquestraDevice(QubitDevice, abc.ABC):
 
     observables = {"PauliX", "PauliY", "PauliZ", "Identity", "Hadamard"}
 
-    def __init__(self, wires, shots=10000, analytic=True, **kwargs):
-        super().__init__(wires=wires, shots=shots, analytic=analytic)
+    def __init__(self, wires, shots=None, **kwargs):
```

Comment:
Worth noting, that previously we had every device except for IBMQ be initialized as `analytic=True`. With the current state of the PR, we have:

`shots=None`:
* Qulacs

`shots is not None`:
* Qiskit
* Forest

It makes sense having `Qiskit` and `Forest` that way because they both have the ability to simulate HW devices and their state vector simulators can also be run in non-analytic mode (one exception is `statevector_simulator` where we have an explicit warning).

Reply:
Oh! Should this be changed to revert it how it was?

Reply:
I think it's okay to keep it as is, it's just a matter of which one we'd like to have as default. Also, we have warnings for both Qiskit and Forest when `shots is None`. Not sure how we could decide if it was the user passing `shots=None` or that the default argument for `shots` is being used (which is `None`). In both we'd emit a warning :thinking: